internal/workflow/replicator.go

```diff
@@ -215,6 +238,12 @@ func (w *Replicator) execute(ctx workflow.Context, request *ReplicatorRequest) e
 					return xerrors.Errorf("failed to update watermark: %w", err)
 				}
 			}
+
+			if len(responses) > 0 {
+				metrics.Gauge(replicatorHeightGauge).Update(float64(responses[len(responses)-1].LatestBlockHeight))
```

Comment:
we only need the timestamp from the response, because we already know the endHeight, right?

Reply:
Yes. We know the endHeight. Here I want to report the current max height. It can help us to know the progress in the replicate process. Anytime when I check this vaule, I can know the latest height in our storage.

Reply:
I meant the endHeight of the batch, so we don't need to get it from the response.